.github/workflows/publish_releases.yml

```diff
@@ -283,7 +315,7 @@ jobs:
       - uses: ./.github/actions/google-play-common
         with:
           gpc_track: "alpha"
-          secret_arn: "arn:aws:secretsmanager:eu-west-2:308190735829:secret:github-actions/radixdlt/babylon-wallet-android/google-play-console/secrets-OEmdRj"
+          secret_arn: "arn:aws:secretsmanager:eu-west-2:{{ secrets.SECRETS_ACCOUNT_ID }}:secret:github-actions/radixdlt/babylon-wallet-android/google-play-console/secrets-OEmdRj"
```

Comment:
Is this a secret leaked we need to warn the devops team to delete, if they haven't done so yet? @GhenadieVP @sergiupuhalschi-rdx ?